src/components/Home/Sponsors.jsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+import ComponentHeader from "@/UI/ComponentHeader";
+import React, { useEffect, useRef } from "react";
+
+const Sponsors = () => {
+  const scrollContainerRef = useRef(null);
+
+  
+  useEffect(() => {
+    const container = scrollContainerRef.current;
+    let scrollSpeed = 0.5; 
+
+    const scrollLogos = () => {
+      if (container) {
+        container.scrollLeft += scrollSpeed; 
+        if (container.scrollLeft >= container.scrollWidth / 2) {
+          container.scrollLeft = 0; 
+        }
+      }
+    };
+
+    const intervalId = setInterval(scrollLogos, 20); 
+
+    return () => clearInterval(intervalId); 
+  }, []);
+
+  const logos = [
+    "/assets/partners-1.svg", 
+    "/assets/partners-2.svg",
+    "/assets/partners-3.svg",
+    "/assets/partners-4.svg",
+    "/assets/partners-5.svg",
+    "/assets/partners-6.svg",
+  ];
```

Comment:
create a sponsor or partners folder inside assets folder then import the img

Reply:
Done